conf/modules/sonar_vl53l1x.xml

```diff
@@ -27,7 +27,8 @@
   </header>
 
   <init fun="sonar_vl53l1x_init()"/>
-  <periodic fun="sonar_vl53l1x_read()" period="0.020"/>
+  <periodic fun="sonar_vl53l1x_read()" freq="10."/>
```

Comment:
Except that the dataready pin is not available ! Also, the API of Paparazzi for the data ready pins is really bad (i.e. none). So it would end up polling i2c status, which is less efficient than this I think.

Reply:
It would indeed poll over i2c, but is that so much less efficient? Unless there are a lot more sensors connected to the same bus, the i2c peripheral isn't really doing anything else anyway and I don't expect reading/writing to it takes that much time.  If efficiency is a problem, maybe the sys clock can be used to run at a configurable interval instead of a fixed 10Hz? At the very least, it would be nice to increase the frequency to max 50Hz for very agile drones. Anything above that is probably not that useful anymore.